streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala

```diff
@@ -145,6 +146,10 @@ private[ui] class StreamingPage(parent: StreamingTab)
   private val listener = parent.listener
   private val startTime = System.currentTimeMillis()
 
+  parent.parent.attachHandler(
+    StreamingApiRootResource.getServletHandler(parent.parent, listener, startTime)
```

Comment:
the main reason is to maintain the consistency of startTime between the original webUI and the api
maybe it is better to record a start time from the start of streaming context or the listener itself
but that will be a change in definition.